src/App.js

```diff
@@ -63,58 +69,62 @@ function App() {
   return (
     <>
       <ThemeProvider>
-        <GlobalStyle />
-        <Switch>
-          <Route path="/login">
-            <Login />
-          </Route>
-          <Route path="/application/review">
-            <ApplicationReview />
-          </Route>
-          <Route path="/application/confirmation">
-            <ApplicationConfirmation />
-          </Route>
-          <Route path="/application/:part">
-            {params => <ApplicationForm part={params.part} />}
-          </Route>
-          <PageRoute path="/">
-            <Home />
-          </PageRoute>
-          <PageRoute path="/application">
-            <Application />
-          </PageRoute>
-          <PageRoute path="/charcuterie">
-            <Charcuterie />
-          </PageRoute>
-          <PageRoute path="/faq">
-            <Faq />
-          </PageRoute>
-          <PageRoute path="/schedule">
-            <Schedule />
-          </PageRoute>
-          <PageRoute path="/sponsors">
-            <Sponsors />
-          </PageRoute>
-          <PageRoute path="/quicklinks">
-            <Quicklinks />
-          </PageRoute>
-          <PageRoute path="/judging">
-            <Judging />
-          </PageRoute>
-          <PageRoute path="/judging/view/:id">{params => <JudgingView id={params.id} />}</PageRoute>
-          <PageRoute path="/submission">
-            <Submission />
-          </PageRoute>
-          <PageRoute path="/submission/create">
-            <SubmissionCreate />
-          </PageRoute>
-          <PageRoute path="/submission/edit">
-            <SubmissionEdit />
-          </PageRoute>
-          <Route>
-            <Page>Page Not Found!</Page>
-          </Route>
-        </Switch>
+        <AuthProvider>
+          <GlobalStyle />
+          <Switch>
+            <Route path="/login">
+              <Login />
+            </Route>
+            <AuthRoute path="/application/review">
+              <ApplicationReview />
+            </AuthRoute>
+            <AuthRoute path="/application/confirmation">
+              <ApplicationConfirmation />
+            </AuthRoute>
+            <AuthRoute path="/application/:part">
+              {params => <ApplicationForm part={params.part} />}
+            </AuthRoute>
+            <PageRoute path="/">
+              <Home />
+            </PageRoute>
+            <AuthRoute path="/application">
+              <Application />
+            </AuthRoute>
```

Comment:
is it possible to use a subrouter for all `/application` routes?

Reply:
Yes, but it would really only be for organization (I didn't touch the way its organized because i haven't been part of that planning)